### PR TITLE
CMS 151 Create HMS Splash AdView

### PR DESCRIPTION
**Pull request by emineinan**

### CODE REVIEW

File: ads/src/main/java/com/hms/lib/commonmobileservices/ads/splash/CommonSplashAdView.kt [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                    }
+                )
+            }
+
+            MobileServiceType.HMS -> {
+                hmsSplashView = SplashView(context)
+                val adRequestParams = AdParam.Builder().build()
+                hmsSplashView.load(
+                    hmsAdUnitId ?: "",
+                    screenOrientation,
```
Comment (Feyzarkt) [inline ads/src/main/java/com/hms/lib/commonmobileservices/ads/splash/CommonSplashAdView.kt:150]: Is this value found both with the dynamic function and manually from attributes? you must choose one :)

**Review by Feyzarkt — CHANGES_REQUESTED** [review]

File: ads/src/main/java/com/hms/lib/commonmobileservices/ads/splash/CommonSplashAdView.kt (hunk 1 above)
Comment (emineinan) [reply ads/src/main/java/com/hms/lib/commonmobileservices/ads/splash/CommonSplashAdView.kt:150]: ✔

**Review by Feyzarkt — APPROVED** [review]
Great work!

**Review by htk007 — APPROVED** [review]
ok

**Pull request merged**